Docs/user_guide/about.md

```diff
@@ -104,7 +104,7 @@ We invite commercial entities to use 3D Slicer.
 
 - [Ebatinca SL](https://ebatinca.com/) is an international technology company in Las Palmas, Spain focused on technology for sustainable development.
 - [Isomics](http://www.isomics.com/) uses 3D Slicer in a variety of academic and commercial research partnerships in fields such as planning and guidance for neurosurgery, quantitative imaging for clinical trials, clinical image informatics.
-- [Kitware](http://www.kitware.com/opensource/slicer.html) Integral to continuing to support the 3D Slicer community, Kitware is also offering consulting services in response to the rapidly growing demand for the development of proprietary applications and commercial products based on 3D Slicer. Kitware has used 3D Slicer to rapidly prototype solutions in nearly every aspect of medical imaging and is also collaborating on the development of commercial pre-clinical and clinical products based on 3D Slicer.
+- [Kitware](http://www.kitware.com/opensource/slicer.html) has a long history of leading and contributing to open source platforms that serve as the foundation of many medical visualization and data processing applications. Kitware collaborates with customers to meet the demand for the development of proprietary applications and commercial products based on 3D Slicer. The company, whose focus is on solving the world’s most complex scientific challenges through customized software, has used 3D Slicer to rapidly prototype solutions in nearly ever aspect of medical imaging and is collaborating on the development of commercial pre-clinical products based on 3D Slicer.
```

Comment:
These sentences are a bit hard to read, because they are too long, complex, and repetitive. A few suggestions:

- "leading and contributing" - can you choose one (or find a word that covers both)?
- "medical visualization and data processing applications" - can you choose one (or find a term that describes both)?
- "collaborates with customers to meet the demand for the development" -> "helps its customers in developing"
- "proprietary applications and commercial products" - can you choose one (or find a better term)?
- "whose focus is on solving the world’s most complex scientific challenges through customized software" - inserting this in the middle of a sentence makes both this slogan and the sentence hard to grasp. You could maybe start or end the Kitware paragraph with this.
- "has used 3D Slicer to rapidly prototype solutions in nearly every aspect of medical imaging and is collaborating on the development of commercial pre-clinical products based on 3D Slicer" - this is almost the same as the previous sentences (the only new information is that the "pre-clinical" word is specifically mentioned)

Reply:
Thanks for the feedback, I will work with our comm team to update the description

Reply:
Here is an updated version:

> Kitware focuses on solving the world’s most complex scientific challenges through customized software solutions. The company has a long history of contributing to open source platforms that serve as the foundation of many medical visualization and data processing applications. Kitware helps  customers develop commercial products based on 3D Slicer and has used the platform to rapidly prototype solutions in nearly every aspect of medical imaging.